doctor/cores/api_ingress.py

```
@@ -0,0 +1,355 @@
+        Returns:
+            (is_valid, error_message)
+        """
+        # بررسی احراز هویت
+        if not user.is_authenticated:
+            return False, "کاربر احراز هویت نشده است"
+        
+        # بررسی نوع کاربر (اگر مدل User این قابلیت را دارد)
+        try:
+            if hasattr(user, 'user_type') and user.user_type != 'doctor':
+                return False, "کاربر باید پزشک باشد"
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)